lib/member.rb

```diff
@@ -0,0 +1,11 @@
+require 'scraped'
+
+class Member < Scraped::HTML
```

Comment:
Hmm, I don't think it makes sense for `Member` to be a subclass of `Scraped::HTML`. You're quite right to create a `Member` class here - top level concepts like `Member` should be reserved for classes that pull together different concepts.

I think for this we probably want a plain `Member` class without a parent which implements a `#to_h` method which returns the information from the `MemberSection` class with the information from the `MemberPage` class.